paper-ieee-trans.tex

```diff
@@ -426,9 +426,8 @@ \subsubsection{SQA criteria}
 
 \item \textit{Unit and Functional testing}.
 Changes involving the addition of new functionalities, are required to be tested. \textit{Regression
-testing} in this context is accomplished by enforcing the successful execution of the previous
-functionality tests already present in the code to uncover any malfunction introduced by
-the changes. \textit{Unit testing} completes the source code testing evaluation by
+testing} in this context is accomplished by enforcing the definition, and periodic execution, of tests for the fixed issues that ensure these issues are not reintroduced during the development activities.
```

Comment:
"...and periodic execution of the tests covering past fixed issues to ensure that they are not reintroduced during the development activities"

Reply:
agree/like Pablo's rephrashing

Reply:
ok

Reply:
agree

Reply:
ok